### DISCUSSION TITLE
Using singularity with seqr

### DISCUSSION THREAD

**Discussion by lauragails** [broadinstitute/seqr#3329] [Q&A] [answered]
Hello, 
I am excited to try seqr on my data! I have a joint-called vcf in terra, and can use that platform, however it says that it may take ~1 week to load. 

In the meantime, I would like to try on our HPC, however docker is not allowed, and we must use singularity, due to security issues. I am going through the installation instructions, and reading other online tutorials, but also would appreciate guidance if you have any! 

Thank you, 
Laura

**Comment by hanars** [comment, marked as answer]
Hi Laura,

seqr is not supported in singularity, it is only available via docker. I would recommend you wait up to a week for your data to be loaded from Terra, as that will be much faster than getting your own instance running and data loaded. For context, it can takes 2-6 weeks for someone with a strong software background to get seqr up and running locally. 

If you do decide to go with the docker installation, please feel free to reach out with any specific questions/ issues you might have

**Comment by lauragails** [comment]
This context helps immensely - I did switch from our HPC to the google cloud and just started making the docker image. I suspect that I will take your advice here (it makes me feel better!) 

I will message if I have any additional questions.  

**edit** I do see a strange error after I run `docker-compose up -d seqr` on the cloud: 
Creating seqr_postgres_1 ... done
Creating seqr_redis_1    ... done
Creating elasticsearch   ... done

ERROR: for seqr  Container "3218776604d4" is unhealthy.
ERROR: Encountered errors while bringing up the project.

**edit** https://github.com/broadinstitute/seqr/discussions/3242 answers that

**Comment by lauragails** [comment]
Thank you again for answering all of my misc. questions throughout the week! Fortunately my data was picked up on Tuesday, and I'm really excited to get the results. How long do they take on average? I have ~100 individuals in my dataset, and picked a small subset for a test sample. 
 
 (it looks like all trios are blue/being processed, which is great too)

**Reply by hanars** [reply]
So once the data is available in the project (changed from yellow "waiting for data" to blue "in progress") that means all the data is available, and it you can start your analysis! We have some tutorial videos that may be helpful for you if you aren't quite sure how to proceed: https://youtube.com/playlist?list=PLlMMtlgw6qNiY6mkBu111-lpmANKHdGKM
